sympy/physics/mechanics/kane.py

```diff
@@ -55,6 +57,17 @@ class KanesMethod(_Methods):
         Boolean whether the mass matrices and forcing vectors should use the
         explicit form (default) or implicit form for kinematics.
         See the notes for more details.
+    kd_eqs_solver : str, callable
+        Solver to be used for the kinematic differential equations. The default
+        is LU solve. See the notes for more details on what the solver is used
+        for and advise on what solver should be used.
+    constraint_solver : str, callable
+        Solver to be used for the velocity constraints. The default is LU solve.
+        See the notes for more details on what the solver is used for and advise
+        on what solver should be used.
```

Comment:
You need the options here and what the callable should look like.

Reply:
We should also try to make this consistent with supplying a linear solvers to the `rhs()` method. I think you can already do that.

Reply:
> We should also try to make this consistent with supplying a linear solvers to the `rhs()` method. I think you can already do that.

I would rather do this in a separate PR, as we should also consider what would be the best key word argument naming here, as we also have `inv_method`.

Reply:
If we introduce new arguments here, they can clash with what we have. The point would be to see what we have (I think in `A.solve(method=)` and `KanesMethod.rhs()` to see if we can pick good new, common kwargs.